src/app/user/auth/auth/auth.component.ts

```diff
@@ -28,13 +28,12 @@ export class AuthComponent implements OnInit {
     };
 
     this.http.post('http://localhost:3000/logIn', args, { responseType: 'text' }).subscribe((response) => {
-      alert('User Authenticated');
       sessionStorage.setItem('email', this.email);
       sessionStorage.setItem('firstName', response);
       this.router.navigate(['/main']);
     }, (error) => {
       console.log(error);
-      alert(error.error);
```

Comment:
this line is required in case there are errors with the backend, to tell the user about it.

Reply:
I removed it coz it was also appearing whenever there was an error 'User not found' which is unappropriate

Reply:
it is not "inappropriate". It must tell the user "every time" that the user is not in the database. Ideally, this function should be called only after the form data has been validated.

Reply:
> I removed it coz it was also appearing whenever there was an error 'User not found' which is unappropriate

my bad.. this is for the above 'requested changes' comment of yours

Reply:
agreed on this one, btw